src/Connector.sol

```diff
@@ -168,16 +159,14 @@ contract CentrifugeConnector {
         bytes16 trancheId,
         address user,
         uint256 amount,
-        string memory domainName
+        ConnectorMessages.Domain domain
```

Comment:
I don't really like having the Connector smart contract depend on the Messages library. I think it would be good if we can make it so that any changes to messages don't impact the Connector. But we can keep this for v2, when we refactor it, likely with the introduction of a Gateway contract inbetween.

Reply:
That's a good point Jeroen. I suggested to @AStox to move the formatting of the domain to `Messages` to have that function alongside all the other encoding/decoding functions. Let's tackle this when we go for v2 👍